PowerEditor/installer/nppSetup.nsi

```diff
@@ -111,8 +111,40 @@ InstType "Minimalist"
 Var diffArchDir2Remove
 Var noUpdater
 
+!ifdef ARCH64
+; this is needed for the 64-bit InstallDirRegKey patch
+!include "StrFunc.nsh"
+${StrStr} # Supportable for Install Sections and Functions
+!endif
+

```

Comment:
Remove this block please.

Reply:
I have fixed that ARM64 build error, but this cannot be removed from where it is:
https://nsis.sourceforge.io/StrFunc#3.1_How_To_Use_Commands_In_Install_Sections_and_Functions